### DISCUSSION TITLE
How to access host labels in tasks

### DISCUSSION THREAD

**Discussion by exeba** [deployphp/deployer#2613] [Help needed]
I'm trying something like:
` run(' then echo "{{ labels.app }}"');`

The problem is that "labels" is defined inside the Configuration class, but the get function does not check for nested properties.
Is there any way around this?

Thanks

**Comment by hw-dwalter** [comment]
here https://github.com/deployphp/deployer/blob/master/recipe/deploy/info.php#L21 you can see how to access specific label content.
